charts/rstudio-workbench/README.md

```diff
@@ -323,6 +323,7 @@ config:
 | affinity | object | `{}` |  |
 | args | list | `[]` | args is the pod container's run arguments. |
 | command | list | `[]` | command is the pod container's run command. By default, it uses the container's default. However, the chart expects a container using `supervisord` for startup |
+| config.defaultMode | object | `{"jobJsonOverrides":420,"pam":420,"prestart":493,"secret":384,"server":420,"session":420,"sessionSecret":272,"startup":493,"userProvisioning":384}` | defaultMode used for mounting the various configuration configmaps |
```

Comment:
The generated documentation here is really poor -- "object" and numbers that have been translated from Octal -- so maybe we should try to improve it by having per-key docstrings or store the modes as strings.